packages/rest-client-jsonapi/src/services/services.test.ts

```diff
@@ -17,16 +17,27 @@ const Article = {
   },
 } satisfies PartialSchema
 
-// const Feature_Article = {
-//   name: "Article",
-//   type: "Feature_Article",
-//   namespace: "Feature",
-//   displayAttribute: "name",
-//   attributes: {
-//     author: string({ required: true }),
-//     tag: string({ required: true }),
-//   },
-// }
+const Feature_Article = {
+  name: "Article",
+  type: "Feature_Article",
+  namespace: "Feature",
+  displayAttribute: "name",
+  attributes: {
+    author: string({ required: true }),
+    tag: string({ required: true }),
+  },
```

Comment:
lets add a `belongsTo` case here as well and test against it, just because `hasMany` vs `belongsTo` work a bit differently under the hood; better to be safe :D